ravenframework/JobHandler.py

```diff
@@ -977,6 +975,8 @@ def terminateJobs(self, ids):
       @ In, ids, list(str), job prefixes to terminate
       @ Out, None
     """
+    #XXX terminateJobs modifies the running queue, which
+    # fillJobQueue assumes can't happen
```

Comment:
does this comment need to be adjusted given the changes made in this PR as well as #1879?